pkg/yang/types_builtin.go

```diff
@@ -246,9 +246,13 @@ func (e *EnumType) ValueMap() map[int64]string {
 // refer to either a builtin type or type specified with typedef.  Not
 // all fields in YangType are used for all types.
 type YangType struct {
-	Name             string
-	Kind             TypeKind    // Ynone if not a base type
-	Base             *Type       `json:"-"` // dervied from
+	Name string
+	Kind TypeKind // Ynone if not a base type
+	// Base represents the base Type if this is a derived type
```

Comment:
I still would like these to be short end-of-line comments and anything more complicated put in the description of the struct.

Some something like (not sure on wording) in the struct comment:

```
// ...  The Base field is set to the ultimate base type for derived types.  IdentityBase is the
// base statement of an Identityref type.

```
And then:

```
        Base         *Type `json:"-"` // base type dervied from
        IdentityBase *Identity        // identityref's base

```

in the struct itself.